RunOnLogin/Explainer.md

```diff
@@ -1,174 +1,174 @@
-
-### Security
-We will post a separate markdown document for [W3C TAG's Security and Privacy self-review questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/).
-
-## OS Primitives
-While all desktop OSes provide the capability to register an application to be automatically executed with OS login, the specific implementation varies across each, the following is a summary of our current research.
-
-On **macOS**, it is possible to get the auto-start behavior by registering application at *Login Items* on current user profile to run on user login.
-
-On **Linux**, auto-starting an application on login greatly depends on the graphical environment used, additional research is required to determine the options available by environment, suggestions from the community are more than welcome!
-
-One possible option to explore is using the `~/.config/autostart/` directory as specified by the [freedesktop.org/XDG Desktop Application Autostart Specification](https://specifications.freedesktop.org/autostart-spec/autostart-spec-latest.html).
-
-On **Windows**, there are multiple ways to register a native application to be executed on user login, these are:
-1.	Register a new key in the Windows Registry with the application to launch.
-2.	Use scheduled tasks.
-3.	Add a shortcut to the Startup folder in the Start menu.
-
-## Open Questions
-- Is asking for permissions at install time a good choice?
-    - We are working on using the PWA installation prompt to ask for a batch of permissions based on our user study. However, there is a question whether this approach would have any negative impact on the PWA installation funnel.
-- Should the web app be provided with some indicator to signal it was launched via the Run on OS Login capability?
-- Limiting the API usage to only installed web apps results in a rather uncommon permission flow: Going from denied (not an installed web app), to prompt (web app is now installed) then granted (user accepts the prompt). Should there be a change to this uncommon flow while still restricting the API usage to installed web apps only?
+# Run on OS Login
+Authors: [Carlos Frias](https://github.com/carlosfriasand), [Jungkee Song](https://github.com/jungkees)
+
+## Status of this Document
+This document is intended as a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
+* This document status: **Archived**
```

Comment:
Withdrawn is probably better for this - there is no api exposure, and it was only implemented as a user setting